### PR TITLE
minor typo fixes

### PR DESCRIPTION
**Pull request by therealrinku**
## What kind of change does this PR introduce?

just minor typo fixes, nothing crazy  -_-

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/lib/constants/prism.ts
Comment (icarusgk) [inline studio/lib/constants/prism.ts:32]: Good PR 🎯 just a small suggestion for clarity - could you update the phrasing from "add this to support" to "supports class names"? It reads a bit smoother and is more concise.

**Comment by icarusgk** [comment]
Great! 😄 Now let's wait for a review from a frontend team member.

File: studio/lib/constants/prism.ts
Comment (therealrinku) [reply studio/lib/constants/prism.ts]: sure . done !!

**Review by Isaiah-Hamilton — APPROVED** [review]

**Pull request merged**
